### PR TITLE
update GraphQL API to remove unneeded namespace param, add applyCampaign test

### PR DESCRIPTION
**Pull request by sqs**
Just a small contribution that addresses an issue (needless namespace param) raised by @mrnugget and adds a new resolver test.

### CODE REVIEW

**Comment by keegancsmith** [comment]
any chance this could break src-cli users? If so, is that fine? (I can imagine the answer is yes).

**Comment by sqs** [comment]
Yes, but this is targeting a campaigns WIP branch that will be worked on a lot more before merging into master. The CLI will be updated before merging this to master.

File: enterprise/internal/campaigns/resolvers/resolver.go [hunk 1]
```diff
@@ -175,52 +175,20 @@ func (r *Resolver) ChangesetSpecByID(ctx context.Context, id graphql.ID) (graphq
...
-	err = svc.CreateCampaign(ctx, campaign)
-	if err != nil {
-		return nil, err
-	}
 
-	// TODO: This mutation is not done.
-	return &campaignResolver{store: r.store, httpFactory: r.httpFactory, Campaign: campaign}, nil
+	// TODO(sqs): Implement createCampaign when we've implemented applyCampaign and are happy about
+	// how it works.
+	return nil, errors.New("createCampaign is not yet implemented (use applyCampaign instead)")
```
Comment (eseliger) [inline enterprise/internal/campaigns/resolvers/resolver.go:186]: I could imagine this code won't change, but only the code in the service layer, so not sure we need to get rid of it here, but your call

**Comment by mrnugget** [comment]
Sweet! Thanks!

**Pull request merged**